pages/faqs/faq-inspector.mdx

```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
 ### What kind of alerts can I set, if any?
 
 You can receive email alerts when new issues are detected. Head to the Issues dashboard on avo.app to configure email alerts.
+
+### When will the "bubble" from the In-App Inspector (aka Avo Debuggers) be visible in my app?
+
+The "bubble" of the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers) is visible when Avo is used in a `dev` environment. The details differ between platforms as follows:
+
+On Android and iOS the bubble will be shown if you unclude the Inspector SDK and initialize it in the `dev` environment.
+On Web the bubble will be shown if you are using Avo Functions in the `dev` environment.
+On react Native the bubble is not shown automatically. You would have to trigger it manually.
```

Comment:
```suggestion
On React Native the bubble is not shown automatically. You will have to trigger it manually.
```

Reply:
Maybe add a "Learn more about how to control the bubble in React Native" link here?